Frameworks/CoreFoundation/Base.subproj/CFRuntime.c

```diff
@@ -275,6 +275,10 @@ void _CFRuntimeBridgeTypeToClass(CFTypeID cf_typeID, const void *cls_ref) {
     __CFUnlock(&__CFBigRuntimeFunnel);
 }
 
+uintptr_t __CFISAForTypeID(CFTypeID typeID) {
```

Comment:
Wait wait wait..
CF_INLINE in a .c file doesn't make sense.
The intent of having it in the header is for every internal CF consumer to inline it -- not just the ones in this file. The compiler can't inline from one .c file to another (or, LTCG can, but it is difficult).

COMDAT folding might help this, but we still have an unnecessary branch to the function impl.

---
In reply to: [79033450](https://github.com/Microsoft/WinObjC/pull/1008#discussion_r79033450) [](ancestors = 79033450,79025442,79022634,78866010,78862337)